.github/workflows/test.yml

```diff
@@ -44,4 +44,8 @@ jobs:
       - name: Lint
         run: npm run lint
       - name: Test
-        run: npm run test
+        run: npm run test:unit
+      - name: Integration
+        run: npm run test:integration:ci
```

Comment:
Seems unnecessary to add in the `ci` part.  Are you planning on different integration tests per environment?

Reply:
the `:ci` script doesn't set the ENV vars (its set by the GitHub actions). the `test:integration` script sets the env var for the config file and then calls the `:ci` script.

I may be able to dump this if we are no longer using experimental modules, as that feature required some work arounds for importing the jest-puppeteer config file.